static/app/components/charts/groupStatusChart.tsx

```diff
@@ -47,14 +47,14 @@ function GroupStatusChart({
     emphasisColors: [string] | undefined;
     series: Series[];
   }>(() => {
-    if (!stats || !stats.length) {
```

Comment:
If biome was doing `prefer-optional-chain` before, then why was this statement allowed before this commit?

Reply:
it could be that eslint does just the job a bit better than biome, covering more cases 🤷

Reply:
that could be!
then how about slap this rule into eslint first, getting more coverage that way is orthogonal to the rest of the PR

Reply:
oh, but it's type-checked too.

Reply:
yep, it needs type info :/